docs/old/mixins.md

```diff
@@ -9,38 +9,8 @@ Each mixin provides:
 * typescript interface to implement
 * props interface
 
-# how to use
```

Comment:
the example here for using mixins was using the old globalId implementation. probably need to consider making a new example using a relevant feature, or because this is under /old then no harm is done?